geopandas/base.py

```diff
@@ -4104,6 +4104,103 @@ def sample_points(self, size, method="uniform", seed=None, **kwargs):
 
         return GeoSeries(result, name="sampled_points", crs=self.crs, index=self.index)
 
+    def polygonize(self, node=True, full=False):
+        """Creates polygons formed from the linework of a GeoSeries.
+
+        Polygonizes the GeoSeries that contain linework which represents the
+        edges of a planar graph. Any geometry type may be provided as input; only the
+        constituent lines and rings will be used to create the output polygons.
+
+        Lines or rings that when combined do not completely close a polygon will result
+        in an empty GeoSeries. Duplicate segments are ignored.
```

Comment:
```suggestion
        Lines or rings that when combined do not completely close a polygon will be ignored. Duplicate segments are ignored.
```
I don't know if there's a better way of putting it, but if you have incomplete lines or rings and complete lines or rings you won't get an empty GeoSeries